<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>lyricItem</class>
 <widget class="QWidget" name="lyricItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>74</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> QWidget#lyricItem
       {color: silver;
         background-image:url(:/icons/texture.png), linear-gradient(hsla(0,0%,32%,.99), hsla(0,0%,27%,.95));
        selection-background-color:#3d8ec9;
        selection-color: black; background-clip: border;
        border-image: none; outline: 0;}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="cover">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="url"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="showLyrics">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>showLyrics</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icons.qrc">
       <normaloff>:/icons/others/eye.png</normaloff>:/icons/others/eye.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
